--- libmysofa-0.7/libmysofa.pc.cmake.orig	2019-03-31 17:16:24.000000000 +0200
+++ libmysofa-0.7/libmysofa.pc.cmake	2019-07-23 06:26:11.184737290 +0200
@@ -3,8 +3,8 @@
 Version: @PROJECT_VERSION@
 Requires: @PKG_CONFIG_REQUIRES@
 prefix=@CMAKE_INSTALL_PREFIX@
-includedir=${prefix}/include
-libdir=${prefix}/lib
+includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
 Libs: -L${libdir} -lmysofa
 Cflags: -I${includedir}
 
--- libmysofa-0.7/CMakeLists.txt.orig	2019-03-31 17:16:24.000000000 +0200
+++ libmysofa-0.7/CMakeLists.txt	2019-07-23 06:33:15.395772475 +0200
@@ -15,8 +15,8 @@
 SET(CPACK_GENERATOR "DEB")
 SET(CPACK_DEBIAN_PACKAGE_MAINTAINER "IOhannes m zmölnig")
 SET(CPACK_PACKAGE_VERSION_MAJOR "0")
-SET(CPACK_PACKAGE_VERSION_MINOR "5")
-SET(CPACK_PACKAGE_VERSION_PATCH "1")
+SET(CPACK_PACKAGE_VERSION_MINOR "7")
+SET(CPACK_PACKAGE_VERSION_PATCH "0")
 SET(CPACK_DEBIAN_PACKAGE_DEPENDS "zlib1g")
 
 SET(PKG_CONFIG_PRIVATELIBS "")
@@ -67,5 +67,5 @@
 )
 install(FILES
   "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}.pc"
-  DESTINATION "lib/pkgconfig"
+  DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig"
 )
